resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackendUtil.scala

```diff
@@ -34,11 +34,11 @@ import org.apache.spark.internal.Logging
  */
 private[mesos] object MesosSchedulerBackendUtil extends Logging {
   /**
-   * Parse a comma-delimited list of volume specs, each of which
+   * Parse a list of volume specs, each of which
    * takes the form [host-dir:]container-dir[:rw|:ro].
    */
-  def parseVolumesSpec(volumes: String): List[Volume] = {
-    volumes.split(",").map(_.split(":")).flatMap { spec =>
+  def parseVolumesSpec(volumes: Seq[String]): List[Volume] = {
```

Comment:
@HeartSaVioR .
Why do we change this function signature silently in this `Use ConfigEntry ...` PR?

Reply:
This is only used for parsing config value, and we are changing the type of config to `Seq[String]` instead of comma-separated `String`. Same applies for below methods as well.

Similar change was suggested while working on ConfigEntry for others (don't remember which one was...).

I'm not sure changing them would hurt because the object is `mesos` package private.

Reply:
@HeartSaVioR . You should mention that in the PR description.

Reply:
TBH I'm not sure we're requiring to mention every incompatible changes for non-public API, but I'm not strongly against it. I'll add it to the PR description.

Please consider that dealing with arbitrary reviewers who have different views on reviewing is not easy. Similar change was done in other PR without any mention and comment. I'd be happy if we have some rules/policies which are across the community in project, and if it doesn't fall into the case, ideally I'd like to see reviewers just kindly and politely suggest/ask it.

Reply:
My question was about the **mismatch** between PR title and content itself. If you feel like that, sorry about that. I must be careful. Maybe, `silently` was too strong word?
> Why do we change this function signature silently in this `Use ConfigEntry ...` PR?

BTW, why do you think my comment is about incompatible changes?
> I'm not sure we're requiring to mention every incompatible changes for non-public API

Reply:
> My question was about the **mismatch** between PR title and content itself.

I didn't know the reason of the question, and `why` without reason behind makes me feeling that I got non-positive reviews. We would have better understanding of each other when we try to ensure explaining rationalization. 

Btw, the rationalization of method signature change is follow: at the first time I thought I have to add new methods with different signatures (maybe overloaded) but another thought in my mind it would end up letting old methods being not used anymore, and reviewers would ask me to remove them. So flatten the steps into one would be changing signature anyway.

Reply:
Part of the purpose of these small comments, which can vary across reviewers, is that it's a way for all of us to sync on how to handle the small things.

The change to the method seems fine, and isn't part of any API. I don't think it needs to be part of the title as it is in the description, even. Let's conclude there.